draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -1006,14 +1006,14 @@ handshake is as follows:
   of a HMAC code created by using token and nonce received with MP_JOIN(A) as message
   and the derived key described in {{MP_KEY}} as key:
 
-  MP_HMAC(A) = HMAC-SHA256(Key=d-key(B), Msg=TB+RA)
+  MP_HMAC(B) = HMAC-SHA256(Key=d-key(B), Msg=RA+RB)
```

Comment:
I think we should keep the logic of the d-key (i.e. local+remote) for the Msg as well, just to avoid confussion. So, RB+RA for  MP_HMAC(B) and RA+RB for MP_HMAC(A) .... it is also in that way for MP-TCP
```suggestion
  MP_HMAC(B) = HMAC-SHA256(Key=d-key(B), Msg=RB+RA)
```